src/main/java/apoc/mongodb/MongoDBColl.java

```diff
@@ -114,13 +114,20 @@ public long count(Map<String, Object> query) {
     }
 
     @Override
-    public Stream<Map<String, Object>> find(Map<String, Object> query, Map<String, Object> project, Map<String, Object> sort) {
+    public Stream<Map<String, Object>> find(Map<String, Object> query, Map<String, Object> project, Map<String, Object> sort, Map<String, Object> pagination) {
         FindIterable<Document> documents = query == null ? collection.find() : collection.find(new Document(query));
         if (project != null) documents = documents.projection(new Document(project));
         if (sort != null) documents = documents.sort(new Document(sort));
+        if (pagination != null) {
```

Comment:
why not pass two explicit skip / limit values? as parameters that default to 0 and -1 respectively? instead of a map?

Reply:
IMHO, if we pass parameters to the user procedure as a map object, then in the "cypher" it is more readable and compact. And the procedure declaration does not grow.